venvUtils/ensureVenv.py

```diff
@@ -21,6 +21,29 @@
 venv_python_version_path: str = os.path.join(venv_path, "python_version")
 
 
+def verifyExistingVenvPath():
+	"""
+	Verifies that the Python virtual environment at c{VENV_PATH} was actually created at that location,
+	and not just copied or moved there.
+	"""
+	# Activate the Python virtual environement and capture the content of the VIRTUAL_ENV environment variable.
+	existingPath = subprocess.check_output(
+		[
+			os.path.join(venv_path, "scripts", "activate.bat"),
+			"&&",
+			"call", "echo", "%VIRTUAL_ENV%",
+		],
+		shell=True,
+	).decode('utf8').rstrip()

```

Comment:
Windows console isn't Unicode by default. It'll probably work fine as long as there is no characters outside of the ASCI range in the path but to be safe this should be decoded using `oem`